as/cNFT/assembly/types.d.ts

```diff
@@ -7,3 +7,8 @@ export declare type Payout = Map<AccountId, u128>
 
 export declare type Amount = u128
 export declare type Balance = Amount
+
+export declare class NftTransferArgs {
```

Comment:
I would prefer to put only what is actually gonna stay in this file. This one can be moved to `market.ts` for now. Thanks!

Reply:
done